azavea/pwd-fishway
# Style quiz

## Overview
Polish pass of the quiz pages. This PR also adds some additional functionality in the form of a green medallion that encourages the user.

Connects #106, connects #104

### Demo
![Screen Shot 2019-08-02 at 8 38 01 AM](https://user-images.githubusercontent.com/5672295/62373699-5a3fb080-b508-11e9-9260-f1f04ce5e8cf.png)
![Screen Shot 2019-08-02 at 8 38 05 AM](https://user-images.githubusercontent.com/5672295/62373700-5a3fb080-b508-11e9-9363-8868c857c9d4.png)
![Screen Shot 2019-08-02 at 9 14 13 AM](https://user-images.githubusercontent.com/5672295/62373716-63c91880-b508-11e9-934f-ad5704b0189d.png)
![Screen Shot 2019-08-02 at 9 14 17 AM](https://user-images.githubusercontent.com/5672295/62373819-98d56b00-b508-11e9-89ac-ff99ce232617.png)

![Screen Shot 2019-08-02 at 9 14 30 AM](https://user-images.githubusercontent.com/5672295/62373737-6d528080-b508-11e9-8169-d6e5394322c0.png)
![Screen Shot 2019-08-02 at 9 28 59 AM](https://user-images.githubusercontent.com/5672295/62373738-6d528080-b508-11e9-87a6-162e274aebd3.png)

### Notes
- Proportions are not 100% accurate in my screenshots—they are better formatted for the app (probably most notable on the "correct" page, where there is less fish/text overlap).
- It would be nice to have more animations on the "Correct" and "incorrect" screens, but reserving this work as an enhancement for if we have time.


## Testing Instructions
* `git pull`
* To make testing easier, set `isScreensaverVisible: false` and `isQuizVisible: true` in `reducers.js` 
* Play game, getting question right or wrong to see each screen and the navbar fishes change
* Play all the way through the end to see final screen with the bigger medallion
